test/Infer/fshl-reduce-const-shamt.ll

```diff
@@ -0,0 +1,20 @@
+; REQUIRES: solver
+
+; RUN: %llvm-as %s -o - | %souper %solver -souper-infer-inst -souper-synthesis-comps=fshl,const | %FileCheck %s
+
+; XFAIL: *
+; FIXME: does not seem to try to reduce the constant.
```

Comment:
right, souper currently has no preferences among constants -- in general doing a good job at this seems difficult, though in this particular case we might want to add a specific constraint into constant synthesis that makes sure a funnel shift always gets the reduced version of a constant

Reply:
Right. I have looked around in the code, and to me it isn't obvious where that should go, so i think i'll just leave everything as-is for now.